### PR TITLE
Expands partner profile page

### PR DESCRIPTION
**Pull request by csell5**
* Refactors out some orphaned files and eslint issues
* refactors old thatconference.com member reference to that.us/members
* expands partner details page
** adds meet the team section
** updates layout for the goals section
** updates layout for hero.

### CODE REVIEW

**Comment by brettski** [comment]
@csell5 is partner following setup/working?

**Comment by csell5** [comment]
> @csell5 is partner following setup/working?
> 
> 
No not yet. Decided I'd just do that in the next PR today to keep it isolated.

**Comment by csell5** [comment]
> @csell5 is partner following setup/working?
> 
> 
No not yet. Decided I'd just do that in the next PR today to keep it isolated.

File: src/components/activities/KeynoteSpeaker.svelte [hunk 1]
```diff
@@ -31,17 +30,14 @@
...
                 <div>
-
                   <div
                     class="text-center text-3xl leading-9 font-semibold
                     text-gray-900 sm:-mx-6"
                   >
                     <Link
-                      open
-                      href="https://www.thatconference.com/member/{host.profileSlug}"
+                      href="/members/{host.profileSlug}"
```
Comment (brettski) [inline src/components/activities/KeynoteSpeaker.svelte:38]: `host.profileSlug` in one activity component and `speaker.profileSlug` in others?

File: src/routes/partners/_FeaturedMembers.svelte [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      </div>
+      <div class="lg:col-span-2">
+        <ul
+          class="space-y-12 sm:grid sm:grid-cols-2 sm:gap-12 sm:space-y-0 lg:gap-x-8"
+        >
+          {#each members as member}
+            <li class="transform hover:scale-110 cursor-pointer">
+              <Link href="/members/{member.profileSlug}">
+                <div class="flex items-center space-x-4 lg:space-x-6">
+                  <img
```
Comment (brettski) [inline src/routes/partners/_FeaturedMembers.svelte:31]: Some images are being squashed vertically or horizontally. Looking at the member profile page, they layout okay. May want to review the imgix values being used. 

image

File: src/routes/partners/_Goals.svelte [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        >
+          Our Goals
+        </h2>
+      </div>
+
+      <div class="flex flex-col">
+        <ul
+          class="space-y-4 sm:grid sm:grid-cols-2 sm:gap-6 sm:space-y-0 lg:grid-cols-3 lg:gap-8"
+        >
+          {#each goals as goal}
```
Comment (brettski) [inline src/routes/partners/_Goals.svelte:24]: This is fine for now, thought these cards look 'odd' as goals. Perhaps it's formatting.

**Review by brettski — CHANGES_REQUESTED** [review]

File: src/components/activities/KeynoteSpeaker.svelte (hunk 1 above)
Comment (csell5) [reply src/components/activities/KeynoteSpeaker.svelte:38]: in this case, we're taking the export speakers and ripping off the first one in the array calling it host. poorly named sure, but until we support multiples it's something.

File: src/routes/partners/_Goals.svelte (hunk 3 above)
Comment (csell5) [reply src/routes/partners/_Goals.svelte:24]: I honestly don't really have any other ideas at the moment.

File: src/components/activities/KeynoteSpeaker.svelte (hunk 1 above)
Comment (brettski) [reply src/components/activities/KeynoteSpeaker.svelte:38]: My first thought was, multiple speakers in a keynote is not uncommon.... 
makes sense on where things are at.

**Review by brettski — APPROVED** [review]

**Pull request merged**